source/grid/tria.cc

```diff
@@ -1257,7 +1257,18 @@ namespace internal
       << "In SubCellData the line info of the line with vertex indices " << arg1
       << " and " << arg2 << " appears more than once. "
       << "This is not allowed.");
-
+    /**
+     * Exception
+     * @ingroup Exceptions
+     */
+    DeclException2(
+      ExcInconsistentManifoldIdLineInfoOfLine,
+      int,
+      int,
+      << "In SubCellData the line info of the line with vertex indices " << arg1
+      << " and " << arg2
+      << " appears mutiply with different (valid) manifold ids. "
```

Comment:
should be 'multiply' or 'multiple times'. I think 'multiple times' is easier to understand.

Reply:
Well, if a native speaker is confused by "multiply", I am happy to rephrase.